packages/client/src/object/FetchPageArgs.ts

```diff
@@ -20,14 +20,20 @@ import type {
   ObjectOrInterfaceDefinition,
   ObjectOrInterfacePropertyKeysFrom2,
 } from "@osdk/api";
+
+export type StrictNonNull = false | "throw" | "drop";
```

Comment:
This type name confused me a bit at first because it made it sound like it was just enforcing properties on an object or non-null, when in reality its actually looking at the nullability field on each parameter and deciding whether to adhere to it or not. Can we rename to something like `StrictNullability` or basically something that makes it more clear this about whether or not you're respecting the nullabilty on the parameter type?